include/config.h

```diff
@@ -66,6 +70,7 @@
 // 1 is similar to vanilla, but prevents Mario from moving in the wrong direction, and allows finer control with the analog stick.
 // 2 is similar to mode 1, but a bit further from vanilla, and allows instant turnaround if Mario is moving slower than a certain threshold.
 // 3 is instant turning to the intended direction regardless of speed and angle.
+// 4 is an expeimental asymptotic turn.
```

Comment:
```suggestion
// 4 is an experimental asymptotic turn.
```
but do we need an experimental one? i'd think that people could attempt it themselves by tracking down the code, unless you think this feels better